cni-plugin/tests/calico_cni_k8s_test.go

```diff
@@ -1566,7 +1536,7 @@ var _ = Describe("Kubernetes CNI tests", func() {
 
 			// Now create a K8s pod passing in an IP address.
 			name = fmt.Sprintf("run%d", rand.Uint32())
-			pod := ensurePodCreated(clientset, testutils.K8S_TEST_NS, &v1.Pod{
+			ensurePodCreated(clientset, testutils.K8S_TEST_NS, &v1.Pod{
```

Comment:
nit: I thought one of the linters would have gotten angry at this because there's a value returned but no handling for it. I guess if it works, it works.